support-frontend/assets/pages/contributions-landing/contributionsLanding.scss

```diff
@@ -862,6 +862,7 @@ form {
 
 .form__submit .component-button {
   width: 100%;
+  box-sizing: revert;
```

Comment:
potentially `box-sizing: border-box;` may be more clear? I doesn't really matter though!

Reply:
ok, presumably `revert` reverts it to... `border-box`? Then I agree that it's clearer